README.md

```diff
@@ -1,2 +1,44 @@
-# Hello-Microverse
-My first microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Your README file looks great however please stick to the new [template](https://github.com/microverseinc/readme-template) provided by Microverse 👍 Kindly copy all of the content then fill out the sections with information relevant to your project 💯